apps/nestjs-app/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
```

Comment:
ideally this extends the root `tsconfig.options.json`, and we can remove any option here that overlaps

Reply:
Looks like nest is using an older version of TS.

Reply:
You're correct. The command I used, `npx -p @nestjs/cli nest new nestjs-app`, actually sets up the application with NestJS version 9, which is using an older version of TypeScript (4.7.3). We should use the command `npx nest new nestjs-app` instead that utilize NestJS version 10 which is associated with the same version of TypeScript than Moon (5.1.3 actually). Thank you for pointing out the version of TS. I'll update the documentation accordingly!